apps/server/src/modules/authentication/services/authentication.service.spec.ts

```diff
@@ -17,7 +17,7 @@ describe('AuthenticationService', () => {
 	let module: TestingModule;
 	let authenticationService: AuthenticationService;
 
-	let jwtValidationAdapter: DeepMocked<JwtValidationAdapter>;
+	let jwtValidationAdapter: DeepMocked<JwtWhitelistAdapter>;
```

Comment:
```suggestion
	let jwtWhitelistAdapter: DeepMocked<JwtWhitelistAdapter>;
```
Rename `jwtValidationAdapter` to `jwtWhitelistAdapter` to make it more clear what is used here.